lib/ansible/modules/network/slxos/slxos_command.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+#
+# (c) 2018 Extreme Networks Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+from __future__ import (absolute_import, division, print_function)
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: slxos_command
+version_added: "2.6"
+author: "Lindsay Hill (@LindsayHill)"
+short_description: Run commands on remote devices running SLX-OS
```

Comment:
We generally include the company name in the short_description to make it easier for people to find the modules
See http://docs.ansible.com/ansible/latest/modules/list_of_network_modules.html

`Run commands on remote devices running Extreme Networks SLX-OS`